### ISSUE TITLE
Add operator to landuse=cemetery

### ISSUE THREAD

**Issue by maro-21** [openstreetmap/id-tagging-schema#1328]
Add `operator=*` to `landuse=cemetery`.

**Comment by tordans** [comment]
Quick research:
https://taginfo.openstreetmap.org/tags/landuse=cemetery#combinations

<img width="790" alt="image" src="https://github.com/user-attachments/assets/5d83c11d-c07f-4215-9949-b4732eef203c">

Does not look like it is something that is used a lot ATM

**Comment by maro-21** [comment]
Yes, it's not used VERY often because there is no field in the editor. Usage frequency and a field in the editor are mutually dependent.
Anyway, cemeteries has operators.

**Comment by matkoniecz** [comment]
I am also from Poland and here at least it makes perfect sense to record this. Putting it into moreFields should be quite harmless

**Comment by tyrasd** [comment]
I think the addition of the `operator` tag is relatively straight-forward, even though the wiki did not explicitly mention it in the past. It can be useful to distinguish city-run cemeteries ([example](https://www.openstreetmap.org/relation/12286306)) from church-run ones, or to know which church is running which cemetery in big cities. 

> Putting it into moreFields should be quite harmless

That seems like a good approach to me.
